ctl/src/command.rs

```diff
@@ -385,3 +385,68 @@ pub fn status(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>) {
     }
   }
 }
+
+pub fn remove_backend(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, app_id: &str, ip: &str, port: u16) {
+    let instance = Instance {
+      app_id: String::from(app_id),
+      ip_address: String::from(ip),
+      port: port
+    };
+
+    order_command(channel, Order::RemoveInstance(instance));
+}
+
+
+pub fn add_backend(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, app_id: &str, ip: &str, port: u16) {
+  let instance = Instance {
+      app_id: String::from(app_id),
+      ip_address: String::from(ip),
+      port: port
+    };
+
+  order_command(channel, Order::AddInstance(instance));
+}
+
+fn order_command(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, order: Order) {
```

Comment:
I'm not sure factoring the command for `add_backend` and `remove_backend` is a good idea. The `Processing` information when removing a backend server is important, because it will carry how much connections are still active. While adding a backend will be nearly instantaneous.